frontend/src/components/pages/AdminPage.tsx

```diff
@@ -164,7 +199,30 @@ const AdminPage = (): React.ReactElement => {
                   </VStack>
                 </TabPanel>
                 <TabPanel>
-                  <AdminUserTable adminUsers={admins} />
+                  <VStack pt={4} spacing={6}>
```

Comment:
can you refactor the tab panel section to be its own component that accepts either `AdminUserTable` or `TeacherUserTable` as a userTable prop and `admins.length` or `teachers.length` as the count

you may also need to update `SortTablePopover` for teachers to include sorting by `Region`